src/helm/proxy/clients/mechanical_turk_critique_exporter.py

```diff
@@ -112,7 +116,7 @@ def _render_multiple_choice_options_crowd_html(name: str, options: List[str]) ->
 
 def _render_checkbox_options_crowd_html(name: str, options: List[str]) -> str:
     """Render the Crowd HTML for the options of a checkbox question."""
-    return "\n".join(
+    return "<br>\n".join(
```

Comment:
Use `<div></div>` or `<p></p>` instead.

Reply:
Discussed in person. Keep it as is for now.